docs/docs/troubleshooting.md

```diff
@@ -26,8 +26,8 @@ environment variables `CONTRAST_LOG_LEVEL`, `CONTRAST_LOG_FORMAT` and
   format (defaults to `text`).
 - `CONTRAST_LOG_SUBSYSTEMS` is a comma-seperated list of subsystems that should
   be enabled for logging, which are disabled by default. Subsystems include:
-  `snp-issuer`, `kds-getter`, and `snp-validator`. To enable all subsystems, use
-  `*` as the value for this environment variable.
+  `snp-issuer`, `tdx-issuer`, `kds-getter`, `snp-validator` and `tdx-validator`.
+  To enable all subsystems, use `*` as the value for this environment variable.
```

Comment:
I think this is getting confusing. Maybe we can set the attestation type as attribute (`WithAttrs()`) instead of creating new named loggers for each attestation variant?

Reply:
I agree. Should we do this in a separate PR?

Reply:
let's do it in this one, as we must touch the docs anyway.